src/unittest/python/tracker_tests.py

```diff
@@ -0,0 +1,73 @@
+from unittest import TestCase, mock
+
+from vyu import tracker
+
+
+class TestTrackerWaitForFixation(TestCase):
+
+    def setUp(self):
+        self.reader_patch = mock.patch('vyu.tracker.imageio.get_reader')
+        self.timer_patch = mock.patch('vyu.tracker.Timer')
+        self.i2pos_patch = mock.patch('vyu.tracker.image2position')
```

Comment:
This is patched because Elee and I are doing the unit test for this, right?

Reply:
Not really. You should test that functionality separately, but because these are unit tests,  the tracker tests should pass / fail independently of `image2position`.